mutt-trim

```diff
@@ -59,6 +59,8 @@ my @greetouts = (
     '[Aa]bra[cç]os?([,.]|\s*!)?',
     '[Aa]tentamente([,.]|\s*!)?',
     '[Cc]ordialmente([,.]|\s*!)?',
+    'Many thanks([,.]|\s*!)?',
```

Comment:
Thank you very much! Would you mind moving them after line L49 ?

Reply:
Done!